docker-entrypoint.sh

```diff
@@ -2,10 +2,18 @@
 
 [ -z "$LISTEN_PORT" ] && export LISTEN_PORT=8080
 
+[ -z "$CACHE_MAX_SIZE" ] && export CACHE_MAX_SIZE=100m
+
+[ -z "$CACHE_TTL" ] && export CACHE_TTL=24h
+
+[ -z "$CACHE_KEYS_ZONE_SIZE" ] && export CACHE_KEYS_ZONE_SIZE=10m
+
+[ -z "$CACHE_KEY_EXPIRATION_TIME" ] && export CACHE_KEY_EXPIRATION_TIME=60m
```

Comment:
expiration really reminds to TTL. I'd use the same word as in the `proxy_cache_path` 

```suggestion
[ -z "$CACHE_KEY_INACTIVE_TIME" ] && export CACHE_KEY_INACTIVE_TIME=60m
```

and change things below accordingly.

Reply:
done